src/arduino/exampleProvider.ts

```diff
@@ -43,10 +43,18 @@ export class ExampleProvider implements vscode.TreeDataProvider<ExampleItem> {
         this._exmaples = null;
         this._exampleManager.loadExamples().then((examples) => {
             this._exmaples = examples;
+            if (!this.hasAnyExamples(this._exmaples)) {

```

Comment:
If there aren't any examples, won't this array already be empty?

Reply:
Oh, I see it's more complex of a check than that, ignore.

Reply:
Yep, turns out that the "Built-in examples" node is always inserted, so this check if verifying that there are actually leaves in the tree (which represent real examples).